server/text_generation_server/utils/import_utils.py

```diff
@@ -0,0 +1,16 @@
+import subprocess
```

Comment:
Can't torch provide this info?

Reply:
Yes and that may be better. I will update.